charts/operator/templates/configmap.yaml

```
@@ -0,0 +1,10 @@
+{{- if .Values.localRelease }}
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: wandb-chart-configmap
+data:
+  # this should already be base64 encoded
+  operator.tgz: |
+    {{ .Values.wandbChartBinary }}
```

```suggestion
  wandb-chart.tgz: {{ .Files.Get (.Value.chartPath) | b64enc }}
```

Maybe we could do something like that?